scripts/db_migrator.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, namespace, socket=None):
                      none-zero values.
               build: sequentially increase within a minor version domain.
         """
-        self.CURRENT_VERSION = 'version_4_0_0'
+        self.CURRENT_VERSION = 'version_4_0_1'
```

Comment:
I think we can't take this to 202205 easily. Will there be separate PR for 202205?

Reply:
Yes, I am working on 202205 seperate PR.
Advancing 3_0_5 to 3_0_6 but it looks like it is not working smooth.